libs/martech/martech.js

```diff
@@ -100,20 +100,22 @@ function sendTargetResponseAnalytics(failure, responseStart, timeout, message) {
   const timeoutTime = roundToQuarter(timeout);
   let val = `target response time ${responseTime}:timed out ${failure}:timeout ${timeoutTime}`;
   if (message) val += `:${message}`;
-  window.alloy('sendEvent', {
-    documentUnloading: true,
-    xdm: {
-      eventType: 'web.webinteraction.linkClicks',
-      web: {
-        webInteraction: {
-          linkClicks: { value: 1 },
-          type: 'other',
-          name: val,
+  if(window._satellite && window._satellite.track){
+    window._satellite.track('event', {
```

Comment:
```suggestion
  window._satellite?.track?.('event', {
```

Reply:
@overmyheadandbody -makes sense, will update the request Thanks :)